app/js/actions/careSetting.js

```diff
@@ -26,7 +23,7 @@ const fetchPatientCareSettingLoading = status => ({
 
 const fetchPatientCareSetting = () => (dispatch) => {
   dispatch(fetchPatientCareSettingLoading(true));
```

Comment:
I guess we agreed to replace our loader actions with a generic one?

Reply:
The `Integrate a generic loader` ticket is still in progress and I believe you concerns are being worked on by @fuchodeveloper

Reply:
Okay then

Reply:
Okay then, nice work